package.json

```diff
@@ -23,19 +23,18 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "json-path": "^0.1.3",
-    "validator": "^5.2.0"
+    "jsonpath-plus": "^7.2.0",
+    "validator": "^13.9.0"
   },
   "devDependencies": {
-    "koa": "*",
-    "koa-router": "*",
-    "koa-body": "*",
-    "should": "*",
-    "mocha": "*",
-    "supertest": "^0.13.0",
     "coveralls": "*",
+    "koa": "*",
+    "koa-body": "^4",
+    "koa-router": "^4",
```

Comment:
These are not all the way upgraded and there are in fact breaking changes along the way, namely because older koa middleware was expected to be a generator function though that is deprecated at this point.